spec/fabricators/search_stat_fabricator.rb

```diff
@@ -11,5 +9,9 @@
   top_ad_count { rand(1..5) }
   status { rand(1..3) }
   raw_response { FFaker::HTMLIpsum.body }
-  user_id { demo_user.id }
+  user { User.create(email: 'user@demo.com', password: 'Secret@11') }
```

Comment:
Why don't we do it explicitly as previously? 💭 
```suggestion
  user { demo_user }
```

Reply:
Creating explicit user throws below exception while running tests. However, implicit creation doesn't throw any error. If it's required to use explicit declaration, we will have to find other ways to solve the problem


<img width="1372" alt="Screen Shot 2023-06-13 at 2 34 29 PM" src="https://github.com/mosharaf13/google-scrapper-ruby/assets/12990839/fc15650a-aa56-4acb-ac6b-5e9fb801246b">

Reply:
Sorry about that, I thought we were in the `seeds` file. How about using Fabricate instead of creating a new one manually?

```suggestion
  user { Fabricate(:user) }
```

Reply:
Fixed in 5c5eaad6d8b370373b7f1a7571de437b5adb1fd9